### DISCUSSION TITLE
[UNSTABLE] Reordered crew list reseting to default in singleplayer

### DISCUSSION THREAD

**Discussion by DownloadableContent5021** [FakeFishGames/Barotrauma#14766] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When in-game, you can reorganise the crew list by dragging each crewmate's name, however upon finishing a level, the list is reset in the order of who was hired first. 

This has always happened in multiplayer and isn't a big deal, but in single player, switching between characters by using keybinds is a large part of the game, and organising the crew list so that I know exactly who I'll switch to next is very helpful.
![crewlist](https://github.com/user-attachments/assets/b0254ab0-4c8a-4fb7-a5bd-f2c08aa89d50)

### Reproduction steps

1. Re-order your crew list by dragging their names in the list.
2. Finish the round (by docking / undocking from an outpost).
3. Observe as the crew list is reset to the order in which each crew mate was hired.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

Other

### -

v1.6.10.0 (unstable)

### Which operating system did you encounter this bug on?

MacOS

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/72d8bb7df3372f2a80230c43c5e2d151c67d0b0c (commit in our private repo), fix coming up in the next unstable build.
